zerver/lib/bot_lib.py

```diff
@@ -85,19 +85,19 @@ def identity(self) -> BotIdentity:
     def react(self, message: Dict[str, Any], emoji_name: str) -> Dict[str, Any]:
         return {}  # Not implemented
 
-    def send_message(self, message: Dict[str, Any]) -> None:
+    def send_message(self, message: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
Not sure if we need to return a `dict`, why not just `int`

Reply:
I do that because the protocol is defined as `def send_message(self, message: Dict[str, Any]) -> Optional[Dict[str, Any]]:`

Reply:
Ok we will need to make proper changes soon.